victorLi/assignments/cyclicSort/cyclicSortEasy/cyclicSortEasy.js

```diff
@@ -4,10 +4,19 @@
 
 // Write a function to sort the objects in-place on their creation sequence number in O(n) and without any extra space. For simplicity, let’s assume we are passed an integer array containing only the sequence numbers, though each number is actually an object.
 
-const cyclic_sort = function (nums) {
-  // TODO: Write your code here
```

Comment:
Don't forget to include an overview / pseudocode of your approach and to type the solution twice to help improve muscle memory!